docs/osf-trivia-list/dasharo.md

```diff
@@ -1,58 +1,125 @@
+Therefore, the creation of Dasharo was a move to fulfill that need and
+establish a marketing vehicle to deliver value to customers.
+
+In addition, 3mdeb plans to provide a camp for all coreboot refugees, including
+platforms moved to branches due to the need for code evolution, such as Intel
+[Intel Quark SoC deprecation][intel-quark] and [LEGACY_SMP_INIT &
+RESOURCE_ALLOCATOR_V3][legacy-smp]. We want to provide solutions for those
+requiring long-term maintenance and firmware support. More elaborate
+explanation of our position you can find
 [below](#why-dasharo-team-is-against-moving-code-to-branches-in-coreboot).
 
-Dasharo typically may get support for fully open platforms like
-[Raptor Computing Systems Talos II][raptor] family, [ASUS KGPE-D16][kgpe-d16]
-and other (not so open), but also very new stuff like [MSI Z690A DDR4][msi-z690a].
+Dasharo typically supports fully open platforms like [Raptor Computing Systems
+Talos II][raptor] family, [ASUS KGPE-D16][kgpe-d16], and other which are not as
+open but provide modern computing experience, such as [MSI PRO Z690-A
+DDR4/DDR5][msi-z690a]. The goal is to provide a reliable, secure, and scalable
+firmware solution for a wide range of platforms and applications, aligning with
+the vision of a new golden age of computing advocated by experts in computer
+architecture.
 
 ## What Dasharo provides?
 
-For every supported hardware Dasharo Team provides:
+Dasharo has 10 rules that govern the production and release of firmware within
+its ecosystem. Dasharo rules definte what we deliver with every release. These
+rules are:
+
+1. Every release of firmware produced by Dasharo Ecosystem must contain [source
+code](https://github.com/Dasharo), binary, SHA256 hash, and Dasharo
+cryptographic signature of that hash.
+1. Dasharo Universe contains structured documentation for key activities
+related to open-source firmware life-cycle: initial deployment, update and
+recovery.
+1. Cryptographic keys hierarchy should be followed: 
+
+	- [CEO/Founder](https://github.com/3mdeb/3mdeb-secpack/blob/master/keys/owner-key/piotr-krol-key.asc)
+(GPG fingerint: `E030 9B2D 85A6 7E84 6329  E34B B2EE 71E9 67AA 9E4C`) which
+signs
+	- [3mdeb Master
+Key](https://github.com/3mdeb/3mdeb-secpack/blob/master/keys/master-key/3mdeb-master-key.asc)
+(GPG fingerint: `1B57 85C2 965D 84CF 85D1  652B 4AFD 81D9 7BD3 7C54`) which
+signs
+	- [3mdeb Dasharo Master
+Key](https://github.com/3mdeb/3mdeb-secpack/blob/master/dasharo/3mdeb-dasharo-master-key.asc)
+(GPG fingerint: `0D5F 6F1D A800 329E B7C5  97A2 ABE1 D0BC 6627 8008`) which
+signs
+	- [Customer Open Source Firmware Release x.y Signing
+Key](https://github.com/3mdeb/3mdeb-secpack/tree/master/customer-keys) (e.g.
+Novacustom Open Source Firmware Release 1.0 Signing Key)
+	- or [dedicated 3mdeb
+keys](https://github.com/3mdeb/3mdeb-secpack/tree/master/dasharo) to given
+platform.
+
+	Keys can be found in [3mdeb-secpack](https://github.com/3mdeb/3mdeb-secpack) repository.
```

Comment:
```suggestion
    - [CEO/Founder](https://github.com/3mdeb/3mdeb-secpack/blob/master/keys/owner-key/piotr-krol-key.asc)
(GPG fingerint: `E030 9B2D 85A6 7E84 6329  E34B B2EE 71E9 67AA 9E4C`) which
signs
    - [3mdeb Master
Key](https://github.com/3mdeb/3mdeb-secpack/blob/master/keys/master-key/3mdeb-master-key.asc)
(GPG fingerint: `1B57 85C2 965D 84CF 85D1  652B 4AFD 81D9 7BD3 7C54`) which
signs
    - [3mdeb Dasharo Master
Key](https://github.com/3mdeb/3mdeb-secpack/blob/master/dasharo/3mdeb-dasharo-master-key.asc)
(GPG fingerint: `0D5F 6F1D A800 329E B7C5  97A2 ABE1 D0BC 6627 8008`) which
signs
    - [Customer Open Source Firmware Release x.y Signing
Key](https://github.com/3mdeb/3mdeb-secpack/tree/master/customer-keys) (e.g.
Novacustom Open Source Firmware Release 1.0 Signing Key)
    - or [dedicated 3mdeb
keys](https://github.com/3mdeb/3mdeb-secpack/tree/master/dasharo) to given
platform.

    Keys can be found in [3mdeb-secpack](https://github.com/3mdeb/3mdeb-secpack) repository.
```

mkdocs uses 4 spaces for indents IIRC. Hard TABs should be avoided

Reply:
grrr, my new dev vm setup messed up with editor configuration